src/widget/form/profileform.cpp

```diff
@@ -242,7 +242,8 @@ void ProfileForm::onAvatarClicked()
     // If this happens, you're really doing it on purpose.
     if (bytes.size() > 65535)
     {
-        QMessageBox::critical(this, tr("Error"), tr("This image is too big"));
+        QMessageBox::critical(this, tr("Error"),
+            tr("This image is too big.\nPlease supply image smaller than 64 KiB, or try with different one.\nIf you think that your image should work, please report a bug with it attached."));
```

Comment:
Some english mistakes as far as I can tell:
Please supply image -> Please supply an image
try with different one -> try with a different one

I'm not sure if "If you think that your image should work, please report a bug with it attached." should be in an error message.
This error will probably never happen, so no need to handheld the user here. I'm not convinced that it's technically possible at all to craft a file that would trigger this error. If it happens, then you really tried very hard to make it happen and you can find the bugtracker by yourself :)